src/languages/css-language.js

```diff
@@ -38,6 +39,22 @@ import { visitorKeys } from "./css-visitor-keys.js";
  * @property {SyntaxConfig} [customSyntax] Custom syntax to use for parsing.
  */
 
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+const blockOpenerTokenTypes = new Map([
+	[tokenTypes.LeftCurlyBracket, "}"],
```

Comment:
```suggestion
const blockOpenerTokenTypes = new Map([
	[tokenTypes.Function, ")"],
	[tokenTypes.LeftCurlyBracket, "}"],
```

For functions missing closing `)`.

Example: `a { width: min(40%, 400px; }`

Without this, the error text is `Parsing error: Missing closing undefined`.